iso/create-osrelease.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+VERSION=`cat ../VERSION`
+GITTAG=`git rev-parse --short HEAD`
+TODAY=`date +"%d%m%Y%H%M%S"`
+
+cat > os-release <<-EOF
```

Comment:
@hferentschik Adding os-release file will not help as make_iso.sh in https://github.com/minishift/minishift-b2d-iso/blob/master/iso/Dockerfile#L45 overwrites it. Hence the sed command to remove the code .